File: app/src/main/java/com/appoets/gojek/provider/views/onboard/OnBoardViewModel.java
package com.appoets.gojek.provider.views.onboard;

import com.appoets.base.base.BaseViewModel;

public class OnBoardViewModel extends BaseViewModel<OnBoardNavigator> {

    public void openSignIn() {
        getNavigator().goToSignIn();
    }

    public void openSignUp() {
        getNavigator().goToSignUp();
    }

}

File: app/src/main/java/com/xjek/provider/views/onboard/OnBoardViewModel.kt
package com.xjek.provider.views.onboard

import com.xjek.base.base.BaseViewModel

class OnBoardViewModel (val onboardNavigator: OnBoardNavigator): BaseViewModel<OnBoardNavigator>() {

    fun openSignIn() {
        onboardNavigator.goToSignIn()
    }

    fun openSignUp() {
        onboardNavigator.goToSignUp()
    }

}
